pkg/webui/console/store/actions/applications.js

```diff
@@ -47,6 +47,12 @@ import {
 
 export const SHARED_NAME = APPLICATION
 
+export const CREATE_APP_BASE = 'CREATE_APP'
+export const [
+  { request: CREATE_APP, success: CREATE_APP_SUCCESS, failure: CREATE_APP_FAILURE },
+  { request: createApp, succedd: createAppSuccess, failure: createAppFailure },
+] = createRequestActions(CREATE_APP_BASE, (ownerId, app, isAdmin) => ({ ownerId, app, isAdmin }))
```

Comment:
```suggestion
] = createRequestActions(CREATE_APP_BASE, (ownerId, app, isAdmin) => ({ ownerId, app, isUserOwner }))
```
`isAdmin` does not exist.